src/@types/common.ts

```diff
@@ -20,3 +20,5 @@ export type NonEmptyArray<T> = [T, ...T[]];
 export type Without<T, U> = { [P in Exclude<keyof T, keyof U>]?: never };
 export type XOR<T, U> = T | U extends object ? (Without<T, U> & U) | (Without<U, T> & T) : T | U;
 export type Writeable<T> = { -readonly [P in keyof T]: T[P] };
+
+export type EmptyObject = Record<PropertyKey, never>;
```

Comment:
@t3chguy @andybalaam please can we stop adding types without documentation? It may be "obvious" but spelling things out makes the code easier to grok and it's part of our agreed coding standards.